filepicker/src/main/java/dev/arkbuilders/components/filepicker/ArkFilePickerViewModel.kt

```diff
@@ -180,6 +181,9 @@ internal class ArkFilePickerViewModel(
     }
 
     fun pinFile(file: Path) = intent {
+        Log.d("tuancoltech", "pinFile isDirectory: " + file.isDirectory())
```

Comment:
I think this log can be removed

Reply:
@mdrlzy Please do not mind the changes yet, this is `Work in progress` MR so there will be upcoming changes.
I'll notify when it's ready for review :)